e2e/utils.go

```diff
@@ -32,9 +32,11 @@ const (
 	cephfsNodePluginSecretName  = "rook-csi-cephfs-node"
 	cephfsProvisionerSecretName = "rook-csi-cephfs-provisioner"
 
-	// rook created rbd user
-	rbdNodePluginSecretName  = "rook-csi-rbd-node"
-	rbdProvisionerSecretName = "rook-csi-rbd-provisioner"
+	// Secret created inside the cephCSINamespace, can be modified. The
+	// Rook secrets get reconciled and changes are undone (needed for
+	// encryption).
+	rbdNodePluginSecretName  = "csi-rbd-secret"
+	rbdProvisionerSecretName = "csi-rbd-secret"
```

Comment:
this can be a temporary fix. currently, the ceph admin key is used in `csi-rbd-secret` this might cause an issue when we want to test few scenarios with limited capabilities.

we need to create a new ceph user and use that in secrets.

Reply:
and we need to add "encryptionPassphrase" to the `csi-rbd-secret` without it the encryption will fail.

Reply:
`encryptionPassphrase` is set by default in the Secret, just the StorageClass does not enable encryption by default.

Reply:
Thanks Niels

Reply:
> this can be a temporary fix. currently, the ceph admin key is used in `csi-rbd-secret` this might cause an issue when we want to test few scenarios with limited capabilities.
> 
> we need to create a new ceph user and use that in secrets.

Sure, we could create a copy of the Rook created Secrets, modify them and store them in the namespace where Ceph-CSI is deployed. But currently we already create the Secret from the examples, so it was just easier to use that (it was not used at all, I think).